fragments/app-sso-client/accelerator.yaml

```diff
@@ -1,71 +1,74 @@
-              ["Securing your first Workload" tutorial](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-index.html)
+              ["Securing your first Workload" tutorial](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-app-operators-secure-spring-boot-workload.html)
               .
-  
-              As stated in the mentioned above tutorial a ClientRegistration custom resource definition is a starting point of
-              an AppSSO integration. There is a `client-registration.yaml` file generated in the `config` directory. You can
-              deploy it applying following command in your project root directory.
-              
-              ```bash
-              \$ kubectl apply -f config/client-registration.yaml
-              ```
-              
-              The next step is to deploy a ResourceClaim which will be used to bind your workload with previously
-              created ClientRegistration. There is a correct `client-registration-claim.yaml` file generated for you
-              in the `config` directory. It can be used to deploy this ResourceClaim.
-              
-              ```bash
-              \$ kubectl apply -f config/client-registration-claim.yaml
-              ```
-              
-              Now you are ready to deploy your Workload with an AppSSO integration. Your `workload.yaml` is already prepared
-              and it includes a service claim referencing the ResourceClaim you created before. 
+
+              This repository includes an integration to set up an authentication mechanism with AppSSO.
+
+              ## Getting Started
+
+              1. Discover your `AppSSO` service offerings
+
+                 ```bash
+                 tanzu services classes list
+                 ```
+
+                 If there isn''t one,
+
+                   - ask your service operator `OR`
+                   - [create your own `ClusterUnsafeTestLogin`](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-reference-api-clusterunsafetestlogin.html)
+
+              2. Select the service offering you''d like to connect
+
+              4. Claim an instance from the offering
+
+                 ```bash
+                 tanzu services class-claims create <your-workload> \\
+                   --class <your-selected-appsso-offering> \\
+                   --parameter workloadRef.name=<your-workload> \\
+                   --parameter redirectPaths=''["/login/oauth2/code/<your-workload>"]''
+                 ```
+
+              5. Apply your workload
+
+                 ```bash
+                 tanzu apps workload apply \\
+                   --file config/workload.yaml \\
+                   --namespace <namespace> \\
+                   --yes \\
+                   --tail
+                 ```
               '
```

Comment:
**[thought, non-blocking]:** The enclosing single quotes aren't required to produce valid YAML. It's a block scalar already and therefore string-ish. But maybe this needs to be quoted for something that consumes it downstream? 🤔